src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -1457,6 +1494,1629 @@ class Test : System.Attribute
         }
 
         [Fact]
+        public void Scope_Catch_01()
+        {
+            var source =
+@"
+public class X
+{
+    public static void Main()
+    {
+    }
+
+    bool Dummy(params object[] x) {return true;}
+
+    void Test1()
+    {
+        try {}
+        catch when (TakeOutParam(out var x1) && x1 > 0)
```

Comment:
I apologize for this inconvenience. At the moment, I am simply porting tests that we have for pattern variables. The base line comes from there too, modulo the source line content and the location on the line. I will take this feedback into consideration for non-ported tests, I heard the same feedback from other team members too.